packages/RichTextEditing-Tests.package/RichTextEditorFileManipulationTests.class/instance/testUS84loadFileFromFilesystem.st

```diff
@@ -0,0 +1,6 @@
+acceptance-tests
+testUS84loadFileFromFilesystem
+	self setupFile.
```

Comment:
We could also use the generic `setUp` and `tearDown` methods and call this there and thus not having to do setup and teardown in the testcase itself